MapboxDirections/MBVisualInstruction.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+/**
+ :nodoc:
+ Encompasses all information necessary for creating a visual queue about a given `RouteStep`.
+ */
+@objc(MBVisualInstruction)
+public class VisualInstruction: NSObject, NSSecureCoding {
+    
+    /**
+     :nodoc:
+     Distance in meters from the beginning of the step at which the visual instruction should be visible.
+     */
+    public let distanceAlongStep: CLLocationDistance
+    
+    
+    public let primaryText: String
+
+    /**
+     :nodoc:
+     Most important visual content to convey to the user about the `RouteStep`.
```

Comment:
Indicate that this is the structured representation of the value that’s in the `primaryText` property; likewise for `secondaryTextComponents` and `secondaryText`.